### ISSUE TITLE
codegen web generate handler only returns zip file name and not content

### ISSUE THREAD

**Issue by auahmed** [networknt/light-codegen#112]
If the handler is used as a service and hosted somewhere for people to visit and access via a website, the request should return the actual zip file itself rather then the file name as the user might not have access to the server.
Currently the file is placed into /tmp/zip looking at the default config directory settings.

**Comment by NicholasAzar** [comment]
@stevehu I can take a look at this if no one has already?

**Comment by stevehu** [comment]
@NicholasAzar Yes. Please do. Thanks.

**Comment by auahmed** [comment]
hi @NicholasAzar , was wondering if the issue was fixed?
i had pulled the new code locally and built the project but it seems like sending the request with /api/zip in the path it does not go into the handler. i get no response back and the following is what i see in the console:

````
11:48:53.111 [XNIO-1 task-10]  V1SYtN9zSHKTRGvIxM0Rmg DEBUG io.undertow.request match - Matched default handler path /api/zip
````

wanted to confirm if it was fixed and if possible would you might know why i might not be getting a response.

**Comment by NicholasAzar** [comment]
@auahmed yeah it was working for me when i finished. I think you just need to make sure what you have installed is the latest from develop. Double check to see if your local light, light-hybrid, and light-codegen (in that order) are both on develop with latest pulled, and installed. If that doesn't solve the problem i'll spend more time trying to recreate the issue.

**Comment by auahmed** [comment]
Thanks @NicholasAzar !!! i guess i hadn't built it in the right order and cuz of that it did not work before.
Worked now and everything looks great

**Comment by auahmed** [comment]
@NicholasAzar was there a specific reason you switched from using axios to form for the download button?

**Comment by NicholasAzar** [comment]
@auahmed it was only since I already knew file download from a form and didn't want to risk going down a rabbit hole 😄 
In hindsight though, i should have stuck with an ajax request since i realized too late that the response is not necessarily a file.
I'll be making the change to revert back to axios.

**Comment by stevehu** [comment]
This is released along with version 1.5.11
